lighthouse-extension/test/extension.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const path = require('path');
+const assert = require('assert');
+const fs = require('fs');
+const puppeteer = require('puppeteer');
+
+const lighthouseExtensionPath = path.resolve(__dirname, '../app');
+const config = require(path.resolve(__dirname, '../../lighthouse-core/config/default.js'));
+const lighthouseCategories = Object.keys(config.categories);
+
+const manifestLocation = path.join(lighthouseExtensionPath, 'manifest.json');
```

Comment:
can you pull this stuff into a little named function? might as well create another for putting the orig back.

Reply:
actually if we use mocha to run this test, then we can use before/after methods for these things.

seems fairly reasonable to use mocha at this point

Reply:
any idea how I should name this test? describe and it text😛.

Reply:
describe('lighthouse chrome extension', ...
it('completes an end-to-end run', ...